packages/grid/x-grid-modules/src/components/styled-wrappers/getStyles.ts

```diff
@@ -0,0 +1,212 @@
+      alignItems: 'center',
+      zIndex: 10,
+    },
+    '&  .overlay .content': {
+      flex: 1,
+      display: 'flex',
+      justifyContent: 'center',
+    },
+
+    '&  .columns-container': {
+      position: 'absolute',
+      top: 0,
+      left: 0,
+      right: 0,
+      overflowX: 'hidden',
+      overflowY: 'hidden',
+      display: 'flex',
+      flexDirection: 'column',
+      borderBottom: `1px solid ${borderColor}`,
+      zIndex: 100,
+      backgroundColor: '#f9f9f9',
+      color: theme.palette.text.primary,
+      fontWeight: theme.typography.fontWeightBold,
+      fontSize: theme.typography.fontSize,
+    },
+    '&  .columns-container .MuiDataGrid-col-cell-wrapper': {
+      display: 'flex',
+      width: '100%',
+      alignItems: 'center',
+    },
+    '&  .MuiDataGrid-col-cell': {
+      position: 'relative',
+      display: 'flex',
+      padding: '0 16px',
+    },
+    '&  .MuiDataGrid-col-cell.sortable': {
+      cursor: 'pointer',
+    },
+
+    '&  .MuiDataGrid-col-cell.center': {
+      justifyContent: 'center',
+    },
+
+    '&  .MuiDataGrid-col-cell.right': {
+      justifyContent: 'flex-end',
+    },
+
+    '& .MuiDataGrid-col-cell .title': {
+      textOverflow: 'ellipsis',
+      overflow: 'hidden',
+      whiteSpace: 'nowrap',
+    },
+    '& .MuiDataGrid-col-cell .column-separator': {
+      position: 'absolute',
+      right: '-12px',
+      zIndex: 100,
+      display: 'flex',
+      flexDirection: 'column',
+      justifyContent: 'center',
+    },
+    '& .MuiDataGrid-col-cell .column-separator .icon.separator': {
+      color: '#bdc3c7',
+    },
+    '& .MuiDataGrid-col-cell .column-separator:hover .separator.resizable': {
+      cursor: 'col-resize',
+      color: 'inherit',
+    },
+    '&  .MuiDataGrid-col-cell *': {
+      maxHeight: '56px',
+    },
+    '& .MuiDataGrid-col-cell.checkbox-selection-header-cell .checkbox-input': {
+      padding: '12px',
+    },
+
+    '& .MuiDataGrid-col-cell-wrapper.scroll .MuiDataGrid-col-cell:last-child': {
+      borderRight: 'none',
+    },
+
+    '&  .data-container': {
+      position: 'relative',
+      flexGrow: 1,
+      display: 'flex',
+      flexDirection: 'column',
+    },
+    '&  .window': {
+      position: 'absolute',
+      bottom: '0px',
+      left: '0px',
+      right: 0,
+      overflowX: 'auto',
+    },
+    '&  .window .viewport': {
+      position: 'sticky',
+      top: '0px',
+      left: '0px',
+      display: 'flex',
+      flexDirection: 'column',
+      overflow: 'hidden',
+    },
+    '&  .window .MuiDataGrid-row': {
+      display: 'flex',
+      width: 'fit-content',
+      backgroundColor: '#fff',
+
+      '&:hover': {
+        cursor: 'pointer',
+        backgroundColor: theme.palette.action.hover,
+        // Reset on touch devices, it doesn't add specificity
+        '@media (hover: none)': {
+          backgroundColor: 'transparent',
+        },
+      },
+    },
+    '&  .window .MuiDataGrid-row.odd': {
```

Comment:
```suggestion
    '& .MuiDataGrid-row.Mui-odd': {
```

I wonder how we should handle the pseudo-selector '.odd' here @mnajdova.